src/knx/application_program_object.h

```diff
@@ -7,9 +7,15 @@ class ApplicationProgramObject : public TableObject
 {
   public:
     ApplicationProgramObject(Memory& memory);
+    uint8_t* save(uint8_t* buffer) override;
+    const uint8_t* restore(const uint8_t* buffer) override;
+    uint16_t saveSize() override;
     uint8_t* data(uint32_t addr);
     uint8_t getByte(uint32_t addr);
     uint16_t getWord(uint32_t addr);
     uint32_t getInt(uint32_t addr);
     double getFloat(uint32_t addr, ParameterFloatEncodings encoding);
+    
+	private:
+    uint8_t _programVersion[5] = { 0, 0, 0, 0, 0};
```

Comment:
Is there a reason to keep _programVersion as a member variable? We can save 5 Byte if don't need it ;)

Reply:
OK, changed in new commit.